plugins/library/messagehub.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: messagehub
+short_description: Get messages from message bus.
+version_added: 2.3.0
+author: Jiri Kulda (@Jurisak)
+options:
+  user:
+    description:
+      - Username to use to connect to the message bus.
+    required: true
+
+  password:
+    description:
+      - Password to use to connect to the message bus.
+    required: true
+
+  selector:
+    description:
+      - JMS selector for filtering messages.
+    required: false
+
+  host:
+    description:
+      - Message bus host.
+    required: false
+    default: ci-bus.lab.eng.rdu2.redhat.com
```

Comment:
We should not set a default value and change 'required:' to 'True'. Since the user should be able to provide whichever message bus host to listen on.